datasync/chaindatafetcher/kafka/utils.go

```diff
@@ -96,3 +97,13 @@ func makeBlockGroupOutput(blockchain *blockchain.BlockChain, block *types.Block,
 	r["transactions"] = rpcTransactions
 	return r
 }
+
+func intToBytes(num uint64) []byte {
+	buffer := make([]byte, 8)
+	binary.BigEndian.PutUint64(buffer, num)
+	return buffer
+}
```

Comment:
Why don't you use `common.Int64ToByteBigEndian`?

Reply:
Reflected. a990b9f